<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="firstScreen">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="firstScreenPage1">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="sortingMainLayout_2" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>20</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="sortingMainLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="2" column="0">
            <layout class="QGridLayout" name="sortingPriceLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="sortingPriceLabel">
               <property name="font">
                <font>
                 <family>Trebuchet MS</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Price</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QSlider" name="sortingPriceSlider">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
               <property name="sliderPosition">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="sortingPriceSpinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QGridLayout" name="sortingCameraLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="sortingCameraLabel">
               <property name="font">
                <font>
                 <family>Trebuchet MS</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Camera</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSlider" name="sortingCameraSlider">
               <property name="minimum">
                <number>12</number>
               </property>
               <property name="maximum">
                <number>16</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="sortingCameraSpinBox">
               <property name="minimum">
                <number>12</number>
               </property>
               <property name="maximum">
                <number>16</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="sortingBatteryLayout">
             <item row="1" column="0">
              <widget class="QSlider" name="sortingBatterySlider">
               <property name="minimum">
                <number>1800</number>
               </property>
               <property name="maximum">
                <number>6000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="sortingBatteryLabel">
               <property name="font">
                <font>
                 <family>Trebuchet MS</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Battery</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="sortingBatterySpinBox">
               <property name="minimum">
                <number>1800</number>
               </property>
               <property name="maximum">
                <number>6000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="sortingMainTitle" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="titleMainLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>10</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Trebuchet MS</family>
                  <pointsize>17</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Filters:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="controlsLayout" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="2">
            <widget class="QPushButton" name="applyFiltersButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>30</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>apply</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="sortTypeComboBox">
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="resetFiltersButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>30</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>80</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>833</width>
             <height>500</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>10</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="maximumSize">
        <size>
         <width>1068</width>
         <height>520</height>
        </size>
       </property>
       <widget class="QWidget" name="gridWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>991</width>
          <height>511</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>40</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="horizontalWidget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="screen2BackButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>10</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Trebuchet MS</family>
                   <pointsize>13</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>⟵back</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>220</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="secondScreenImage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>50</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="secondScreenPhoneName">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>21</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Xiaomi redmi 5a</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>60</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="specsMainLabel">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Full specifications:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel1">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel2">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel3">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel4">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel5">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel6">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel7">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="specsLabel8">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="secondScreenBuyButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>30</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>Trebuchet MS</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>buy button</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>350</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sortingBatterySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingBatterySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingCameraSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingCameraSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingPriceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingPriceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingBatterySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingBatterySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingCameraSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingCameraSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingPriceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sortingPriceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
